### ISSUE TITLE
Travis is broken

### ISSUE THREAD

**Issue by schiffner** [mlr-org/mlr-tutorial#28]
Currently Travis fails when installing binary packages.
Apparently, the lib path is not writeable, which gives the warning and then an error in `install.packages` (which then causes the error in `cat` in `update-packages.r`).

Does anyone know what's going on? I can work on it tomorrow afternoon at the earliest.

```
$ curl -L https://raw.githubusercontent.com/mllg/travis-r-tools/master/update-packages.r -o /tmp/update-packages.r
$ Rscript /tmp/update-packages.r
Searching for outdated packages ...
Updating 1 binary packages: pander
Warning in install.packages(req, lib = user.lib) :
  'lib = "/usr/local/lib/R/site-library"' is not writable
Error in cat(list(...), file, sep, fill, labels, append) : 
  argument 1 (type 'list') cannot be handled by 'cat'
Calls: tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous> -> cat
Execution halted
```

**Comment by berndbischl** [comment]
@mllg  wrote this

**Comment by schiffner** [comment]
Fixed now via f24b87024bb4e2f56eccbd42a54db9036828324e. Just a missing `sudo`.

But note that this is also missing in the travis script for mlr and in the README of https://github.com/rdatsci/travis-r-tools!

@mllg said:

> But note that I'm not sure if that calling update-packages.r is still required for travis, I think the
> functionality is now included in devtools.

I looked into it. I found an issue about the Travis installation process (https://github.com/hadley/devtools/issues/974) where they discuss for example a `devtools::update_package` function. And `install` has an `upgrade_dependencies` arg that defaults to `TRUE` since devtools 1.9.1. Apart from that I couldn't find anything that would be directly helpful here and therefore kept update-packages.r.
If anyone knows a better way let me know.
